src/Common/tests/Tests/System/Text/ValueStringBuilderTests.cs

```diff
@@ -152,6 +163,27 @@ public void Insert_IntCharInt_MatchesStringBuilder()
             Assert.Equal(sb.ToString(), vsb.ToString());
         }
 
+        [Fact]
+        public void AsReadOnlySpan_ReturnsCorrectValue_DoesntClearBuilder()
+        {
+            var sb = new StringBuilder();
+            var vsb = new ValueStringBuilder();
+
+            for (int i = 1; i <= 100; i++)
+            {
+                string s = i.ToString();
+                sb.Append(s);
+                vsb.Append(s);
+            }
+
+            var resultSpan = vsb.AsReadOnlySpan();
```

Comment:
We renamed AsReadOnlySpan to AsSpan.

Also, nit: we don't use resultSpan anywhere. Change to:
`var resultString = new string(vsb.AsSpan());`

Reply:
[Done](https://github.com/dotnet/corefx/pull/27250/commits/b1b17d2a0e58e863e86440651b188ac3aab86c77), thanks!